pkg/cache/clusterqueue.go

```diff
@@ -196,6 +198,19 @@ func (c *clusterQueue) updateQuotasAndResourceGroups(in []kueue.ResourceGroup) b
 }
 
 func (c *clusterQueue) updateQueueStatus() {
+	if features.Enabled(features.TopologyAwareScheduling) &&
+		len(c.tasFlavors) > 0 &&
+		len(c.workloadsNotAccountedForTAS) > 0 &&
+		c.isTASSynced() {
+		// There are some workloads which are not accounted yet for TAS.
+		// We re-add them as not the tasCache is initialized (synced).
+		for k, w := range c.Workloads {
```

Comment:
I would love to add some logs here, but it seems we don't add logs in the cache so far. 

I can  open an issue for that, wdyt @tenzen-y @PBundyra ?

Reply:
Opened: https://github.com/kubernetes-sigs/kueue/pull/5324

I would suggest then we cherry-pick both.